js/visearch.js

```diff
@@ -129,26 +130,28 @@
   function callbackWrap(callback, args) {
     callback(args);
 
-    const curUri = new URI(window.location.href);
-    const reqid = getQueryParamValue(curUri, QUERY_REQID);
-    const imName = getQueryParamValue(curUri, QUERY_IMNAME);
-
-    // send out event if the pixel is in place
-    if (reqid && imName && context.vsPlacementLoaded[settings.placement_id]) {
-      sendEvent(RESULT_LOAD, {
-        queryId: getQueryParamValue(curUri, QUERY_REQID),
-        im_name: getQueryParamValue(curUri, QUERY_IMNAME),
-      });
-    }
+    if (args.status === 'OK' && args.results.length > 0) {
+      // send out event if the pixel is in place
+      const metadata = { queryId: args.reqid };
+      if (args.product_info) {
```

Comment:
should get from request param, the `product_info` in the response section is optional and only appear if `return_product_info` is true